content/zh/docs/tutorials/kubernetes-basics/expose/expose-interactive.html

```diff
@@ -0,0 +1,50 @@
+---
+title: 交互式教程 - 暴露您的应用程序
```

Comment:
```suggestion
title: 交互式教程 - 发布您的应用程序
```

虽然应该翻译为`暴露`，但是感觉`发布`更好听一点。

Reply:
仅供讨论